NGitLab/Models/Issue.cs

```diff
@@ -45,6 +45,9 @@ public class Issue
         [DataMember(Name = "updated_at")]
         public DateTime UpdatedAt;
 
+        [DataMember(Name = "due_date")]
+        public DateTime? DueDate;
```

Comment:
You need to fix the warning / error in Visual Studio. The code fix should add a new line in `NGitLab/PublicAPI.Unshipped.txt`